### PR TITLE
Update README to add CI integration

### PR DESCRIPTION
**Pull request by minthigpen**
Remove Private Preview tags 
Add link to Microsoft docs
Add section on compute instance integration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,24 +130,25 @@ After generating the Responsible AI insights, you can view them in your associat
 #### Known limitations of viewing dashboard in AzureML studio
 Due to the (current) lack of active compute, the dashboard in AzureML studio has fewer features than the dashboard generated with the open source package. To generate the full dashboard in a Jupyter python notebook, please download and use our [open source Responsible AI Dashboard SDK](https://github.com/microsoft/responsible-ai-widgets).
```
Comment (riedgar-ms) [inline README.md:131]: Should this line get changed as well?

File: README.md [hunk 2]
```diff
@@ -130,24 +130,25 @@ After generating the Responsible AI insights, you can view them in your associat
...
+Integrated compute instance must be attached to the Responsible AI dashboard to enable full functionality:
+**Error analysis**
+- Setting your global data cohort to any cohort of interest will update the error tree instead of disabling it.
+- Selecting other error or performance metrics is supported.
+- Selecting any subset of features for training the error tree map is supported.
+- Changing the minimum number of samples required per leaf node and error tree depth is supported.
+- Dynamically updating the heatmap for up to two features is supported.
+**Feature importances**
+- An individual conditional expectation (ICE) plot in the individual feature importance tab is supported.
+Counterfactual analysis
```
Comment (riedgar-ms) [inline README.md:142]: Highlight?

File: README.md (hunk 1 above)
Comment (minthigpen) [reply README.md:131]: yep, good catch

File: README.md (hunk 2 above)
Comment (minthigpen) [reply README.md]: done

**Review by riedgar-ms — APPROVED** [review]

**Pull request merged**